content/tutorials/realtime-attendance-system-iot.textile

```diff
@@ -0,0 +1,262 @@
+  <img src="/images/tutorials/iot-attendance/zapier-event-reactor-rule.png" style="width: 100%" alt="Zapier events reactor rule">
+</a>
+
+Next, you'll see a screen to set this reactor rule up. In the URL field, paste the Zapier endpoint that you copied before. You can add in any optional headers if you like, we won't be needing any for this app. Fill in the channel filter field with "standup" as this is the channel we are publishing the messages into. This makes sure that whenever a message is published on this channel, the specified WebHooks endpoint on Zapier will be triggered. Make sure to uncheck the "enveloped" option, as we just want the data payload to go through directly, without any metadata. After this is done, go ahead and create the rule.
+
+<a href="/images/tutorials/iot-attendance/reactor-rule-settings.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/reactor-rule-settings.png" style="width: 100%" alt="New reactor rule settings">
+</a>
+
+The final step before going back to Zapier is to run a quick test to see if everything works fine. Click on "Test rule", then "Run test". If successful, it should say so as shown below:
+
+<a href="/images/tutorials/iot-attendance/test-reactor-rule.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/test-reactor-rule.png" style="width: 100%" alt="Test reactor rule">
+</a>
+
+Now, go back to Zapier to finish setting up the Zap. Click on "Test and continue" to finish up the first part of the Zap. In the second part, we want to send a message to Slack, so choose that app and fill in the details as shown in the screenshot below. Feel free to customise the message as you like.
+
+<a href="/images/tutorials/iot-attendance/set-up-slack-app.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/set-up-slack-app.png" style="width: 100%" alt="Set up Slack app">
+</a>
+
+<a href="/images/tutorials/iot-attendance/customise-slack-message.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/customise-slack-message.png" style="width: 100%" alt="Customize slack messages">
+</a>
+
+That's it, your Zap is all set up. It's time to test it out.
+
+h2(#step5-test-app). Step 5 - Testing out the app and further reading
+
+In this final step, we'll test our application out. So upload the sketch file to your board once again and place the access card near the RFID sensor. This should now result in a message in the Slack channel that you set up as shown below. Simple and easy!
+
+<a href="/images/tutorials/iot-attendance/slack-message.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/slack-message.png" style="width: 100%" alt="Customize slack messages">
+</a>
+
+h3(#further-reading). Further reading
+
+1. Learn more about the "Arduino ecosystem":https://www.arduino.cc/
+2. If you would like to find out more about Ably Reactor features and capabilities, see the "Reactor documentation":https://www.ably.io/reactor.
```

Comment:
Matt tells me this isn't documentation its a landing page, so I changed it to say "Find out more about Ably reactor features and capabilityes"

Reply:
Ah, good catch. Fixed!